code/modules/mob/mob.dm

```diff
@@ -1374,3 +1374,9 @@ GLOBAL_VAR_INIT(mobids, 1)
 		input = capitalize(copytext_char(input, customsayverb+1))
 	return "[message_spans_start(spans)][input]</span>"
 
+/mob/proc/haswings(mob/living/carbon/human/Target)
+	var/obj/item/organ/wings/Wing = Target.getorganslot(ORGAN_SLOT_WINGS)
+	if(Wing == null)
+		return FALSE
+	else
+		return TRUE
```

Comment:
Define this on human or typecheck to see if you're dealing with a human, or if Target is something else.

Reply:
Cant put it in human.dm because its used by life, it can go in living or something but I think the better idea is to typecheck, Ill get it done